src/cmd.rs

```diff
@@ -119,6 +119,71 @@ pub struct Mon {
     pub addr: String,
 }
 
+#[derive(Deserialize, Debug)]
+pub enum HealthStatus {
+    #[serde(rename = "HEALTH_ERR")]
+    HealthErr,
+    #[serde(rename = "HEALTH_WARN")]
+    HealthWarn,
+    #[serde(rename = "HEALTH_OK")]
+    HealthOk,
+}
+
+#[derive(Deserialize, Debug)]
+pub struct ClusterHealth {
+    pub health: Health,
+    pub timechecks: TimeChecks,
+    pub summary: Vec<String>,
+    pub overall_status: HealthStatus,
+    pub detail: Vec<String>,
+}
+
+#[derive(Deserialize, Debug)]
+pub struct Health {
```

Comment:
Renaming `HealthStatus` to `Health` would require renaming this to something, maybe `HealthServices`